client/components/buttons/addButton.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
```

Comment:
this gives me an error right away
```addButton.tsx``` should be ```AddButton.tsx```
But I'm still not sure what really is wrong